### PR TITLE
Added actual QR code reading.

### PR DESCRIPTION
**Pull request by netaneld122**
Added requirements.txt for dependencies.
Added .gitignore.
Updated README according to new usage.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+.pyre/
+
+# pytype static type analyzer
+.pytype/
+
+# Cython debug symbols
+cython_debug/
+
+# Pycharm
+.idea/
```
Comment (yuvadm) [inline .gitignore:141]: This is a huge .gitignore - I'd minimize it to just `*.pyc` and `__pycache__`

File: README.md [hunk 2]
```diff
@@ -8,7 +8,16 @@ As (not really) specified in https://github.com/MohGovIL/Ramzor
 
 A pythonic implementation of the verification process can be found in [`verify.py`](verify.py).
 
-It requires the [`cryptography`](https://pypi.org/project/cryptography/) package to be installed.
+Run the following command to install the dependency packages:
+```bash
+pip install -r requirements.txt
```
Comment (yuvadm) [inline README.md:13]: I recommend upgrading the entire flow to use `pipenv` which is a slightly more modern approach

File: README.md [hunk 3]
```diff
@@ -8,7 +8,16 @@ As (not really) specified in https://github.com/MohGovIL/Ramzor
...
+```
+
+### Usage
+Save your Green Pass as a `.png` file and execute:
+```bash
+python verify.py path_to_my_green_pass_image.png
```
Comment (yuvadm) [inline README.md:19]: We should probably accept both images and pre-decoded payloads

File: requirements.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+cryptography
+pillow
+pyzbar
```
Comment (yuvadm) [inline requirements.txt:3]: Replace with `Pipfile`

File: verify.py [hunk 5]
```diff
@@ -1,28 +1,55 @@
...
+from pyzbar import pyzbar
 
-payload = '{"id":"01/IL/ABCD1234ABCD1234ABCD1234ABCD1234#ABCD1234","et":1,"ct":1,"c":"IL MOH","cn":null,"fn":null,"g":null,"f":null,"gl":null,"fl":null,"idp":null,"idl":null,"b":"0001-01-01","e":"0001-01-01","a":"0001-01-01","p":[{"idl":"0123456789","e":"2021-01-01"}]}'
 
-h = hashes.Hash(hashes.SHA256())
-h.update(payload.encode("utf-8"))
-digest = h.finalize()
+def cert(name):
+    this_dir = os.path.dirname(os.path.abspath(__file__))
+    return os.path.join(this_dir, 'certs', name)
```
Comment (yuvadm) [inline verify.py:16]: A nicer approach would be to use `pathlib` for concat'ing paths

**Review by yuvadm — CHANGES_REQUESTED** [review]
Thanks for the contribution @netaneld122 ! Added some comments

File: .gitignore (hunk 1 above)
Comment (netaneld122) [reply .gitignore:6]: Fixed

File: README.md (hunk 2 above)
Comment (netaneld122) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (netaneld122) [reply README.md]: Fixed

File: requirements.txt (hunk 4 above)
Comment (netaneld122) [reply requirements.txt]: Fixed

File: verify.py (hunk 5 above)
Comment (netaneld122) [reply verify.py]: Fixed

**Review by yuvadm — APPROVED** [review]
Looks good! Some small changes needed but I'll handle those after merging. Thanks @netaneld122 :pray:

**Pull request merged**